skt/runner.py

```diff
@@ -96,9 +96,10 @@ def getresultstree(self, jobid, logs=False):
         Retrieve Beaker job results in Beaker's native XML format.
 
         Args:
-            jobid: The beaker jobid
-            logs:  Set to 'True' to retrieve logs from beaker. The default is
-                   'False'.
+            jobid: The Beaker jobid
+            logs:  Set to 'True' to retrieve logs from Beaker as part of the
+                   job results. The default is 'False', which excludes logs
+                   from the Beaker job results.
```

Comment:
I think this should go into the previous commit.

Reply:
Fixed.